File: src/main/java/it/ipzs/pidprovider/dto/ParResponse.java
package it.ipzs.pidprovider.dto;

import java.io.Serializable;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Data;

@Data
public class ParResponse implements Serializable {

	private static final long serialVersionUID = -883374915611532910L;

	@JsonProperty("request_uri")
	private String requestUri;

	@JsonProperty("expires_in")
	private int expiresIn;

}

File: sdk-android/androidpidprovider/src/main/java/it/ipzs/androidpidprovider/network/response/ParResponse.kt
@file:Suppress("unused")

package it.ipzs.androidpidprovider.network.response


import com.google.gson.annotations.SerializedName
import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import java.io.Serializable

@Keep
internal class ParResponse: Serializable{

    @SerializedName("request_uri")
    @Expose
    val requestUri: String? = null

    @SerializedName("expires_in")
    @Expose
    val expiresIn: Int? = null

}